lists-service/helm/config/lists-service-config.properties

```diff
@@ -0,0 +1,49 @@
+app.name=species-lists
+app.description=${app.name} is a Spring Boot application
+spring.graphql.graphiql.enabled=true
+spring.graphql.graphiql.path=/graphiql
+elastic.host=elasticsearch:9200
+springdoc.swagger-ui.path=
+namematching.url=http://name-matching-service:443
+release.directory=/tmp/
+release.s3.bucket=s3://lists/releases
+release.s3.enabled=false
+spring.servlet.multipart.max-file-size=10MB
+spring.servlet.multipart.max-request-size=10MB
+
+spring.data.mongodb.uri=mongodb://${mongodb.username}:${mongodb.password}@${mongodb.host}:${mongodb.port}/?tls=true&retryWrites=false
+spring.data.mongodb.database=${mongodb.database}
+
+bie.url=https://bie-ws.ala.org.au/ws/species/%s
+bie.images.url=https://bie.ala.org.au/ws/imageSearch/%s?rows=%s&start=%s
+image.url=https://images.ala.org.au/image/%s/large
+
+security.jwt.userIdClaim=username
+security.jwt.roleClaims=cognito:groups
+security.jwt.requireUserInfo=true
+security.jwt.userProfileFromAccessToken=true
+security.jwt.rolesFromAccessToken=true
+
+security.jwt.enabled=${oidc.enabled}
+security.jwt.discovery-uri=${oidc.discoveryUri}
+security.jwt.clientId=${oidc.clientId}
+security.apikey.enabled=false
+
+# admin role and admin scope
+security.admin.role=ROLE_ADMIN,ala/internal
+
+migrate.url=https://lists-test.ala.org.au
+
+# userdetails
+userDetails.url=https://api.test.ala.org.au/userdetails/cognito
+userDetails.api.url=https://api.test.ala.org.au/userdetails/cognito
+
+# validation constraints
+constraints.file=/data/lists-service/config/constraints.json
+
+mongodb.host={{ .Values.mongodb.host }}
+mongodb.port={{ .Values.mongodb.port }}
+mongodb.database={{ .Values.mongodb.database }}
+
+oidc.enabled={{ .Values.oidc.enabled }}
+oidc.discoveryUri={{ .Values.oidc.discoveryUri }}
```

Comment:
All config values are in config.ini, this is still a bit of a work in progress but we have a [working example for userdetails ](https://github.com/AtlasOfLivingAustralia/userdetails/blob/feature/containerization/cicd/deploy/config.ini)